src/common/dto/article/create.dto.js

```diff
@@ -10,6 +10,12 @@ class CreateArticleInputDto {
     }).options({ abortEarly: false });
 
     constructor(data) {
+        if (typeof data.tags == "string") {
+            data.tags = data.tags.split(",");
+        } else {
```

Comment:
This else might not be necessary if we go for not making tags a required field

Reply:
I agree with not making tags a required field

Reply:
The method used to allow for an empty Tags field was:
Allow an empty string on joi (allow.(''))
Check if it's empty and, if so, make it into an array
Else it'll split